pkgs/build-support/trivial-builders.nix

```diff
@@ -559,10 +559,22 @@ rec {
    *                 passthru.tests.greeting = callPackage ./test { };
    *               } ./myscript.sh;
    */
-  makeSetupHook = { name ? "hook", deps ? [], substitutions ? {}, meta ? {}, passthru ? {} }: script:
+  makeSetupHook =
+    { name ? "hook"
+    , substitutions ? { }
+    , meta ? { }
+    , passthru ? { }
+    , deps ? [ ]
```

Comment:
Why change the ordering? It does not appear to be alphabetical nor does it follow the [common order for `mkDerivation`](https://github.com/jtojnar/nixpkgs-hammering/blob/1b038ef38fececb39b65a4cdfa7273ed9d9359b4/explanations/attribute-ordering.md).